Nc-News/src/App.jsx

```diff
@@ -13,6 +14,7 @@ function App() {
         <Route path="/" element={<Home />} />
         <Route path="/articles" element={<Articles />} />
         <Route path="/articles/:article_id" element={<Article />} />
+        <Route path="/articles/:article_id/comments" element={<Comments />} />
```

Comment:
Fine if you want to keep this but consider if you need your comments on a separate page to your article, or if a user may want to view them on the same page as an article's content 🔍